src/modules/ekf2/ekf2_params.c

```diff
@@ -1117,15 +1117,34 @@ PARAM_DEFINE_FLOAT(EKF2_MAGBIAS_Z, 0.0f);
 PARAM_DEFINE_INT32(EKF2_MAGBIAS_ID, 0);
 
 /**
- * State variance assumed for magnetometer bias storage.
- * This is a reference variance used to calculate the fraction of learned magnetometer bias that will be used to update the stored value. Smaller values will make the stored bias data adjust more slowly from flight to flight. Larger values will make it adjust faster.
+ * State variance for magnetometer x-axis bias storage
+ * This is a reference variance used to calculate the fraction of learned magnetometer bias that will be used to update the stored value.
+ *
+ * @group EKF2
+ * @unit mgauss^2
```

Comment:
```suggestion
 * @unit mgauss^2
 * @volatile
```

Any parameters that are updated by the system in normal usage like this should be marked to volatile. This excludes them from the parameter hash used by QGC and can prevent having to resync the entire parameter set every new session.